pandas/tests/io/parser/test_common.py

```diff
@@ -1351,6 +1351,30 @@ def test_numeric_range_too_wide(all_parsers, exp_data):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("neg_exp", [-617, -100000, -99999999999999999])
+def test_very_negative_exponent(all_parsers, neg_exp):
+    # GH#38753
+    parser = all_parsers
+    data = f"data\n10E{neg_exp}"
+    for precision in parser.float_precision_choices:
```

Comment:
you can parameterize on these too

Reply:
Issue I ran into is that the parsers don't have consistent potential `float_precisions`, eg `PythonParser` only has `None`.  Could parameterize and just assert failure on disallowed combinations of `float_precision` and `parser`, but that seems ugly. Will try to think of a better solution.

Reply:
Some kind of fixture-within-a-fixture functionality would be nice, though not sure if that's feasible...

Reply:
Added a new fixture, `all_parsers_all_precisions`. Not the most elegant solution, essentially just precomputes all allowed combos and uses that as the source for the fixture. If you have a better idea please let me know!